Gems/Atom/Bootstrap/Code/Source/BootstrapSystemComponent.cpp

```diff
@@ -322,10 +322,19 @@ namespace AZ
             {
                 // Create a native window only if it's a launcher (or standalone)
                 // LY editor create its own window which we can get its handle through AzFramework::WindowSystemNotificationBus::Handler's OnWindowCreated() function
+
+                // Query the application type to determine if this is a headless application
                 AZ::ApplicationTypeQuery appType;
                 ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationBus::Events::QueryApplicationType, appType);
-                if (appType.IsHeadless())
+
+                // Check for any Registry Setting override for the flag to create the native window or not. 
+                const auto* settingsRegistry = AZ::SettingsRegistry::Get();
+                if (settingsRegistry)
                 {
+                    settingsRegistry->Get(m_createNativeWindow, "/O3DE/Atom/Bootstrap/CreateNativeWindow");
+                }
+
+                if (appType.IsHeadless() || !m_createNativeWindow )                {
```

Comment:
The curly brace should be on the next line.

Reply:
+ remove space before the )

Reply:
Yeah, this PR is still in draft, and might change drastically as I try out different approaches